### PR TITLE
Add Dark and System themes

### PR DESCRIPTION
**Pull request by ilyapopovs**
Add Dark and System themes.
System theme switches between Light and Dark based on `prefers-color-scheme` media feature.

Changes sum-up:
- add new Settings entry
- replace hard-coded CSS colors with CSS Variables (except ones that can stay the same)
- switch CSS Variable values based on the parent class
- add the according class to `body`

### CODE REVIEW

File: src/popup/App.scss [hunk 1]
```diff
@@ -1,262 +1,299 @@
...
-    .dashboard-btn {
-      height: 20px;
-      margin-top: 2px;
-      padding: 0 0.5rem !important;
-    }
-  }
-}
+@import "./App-color.scss";
+@import "./App-scrollbar.scss";
+@import "./Theme.scss";
```
Comment (ilyapopovs) [inline src/popup/App.scss:3]: The whole file is marked as changed due to line endings, changed from CRLF to LF.
Let me know if it's significant and you'd like me to revert them.

**Comment by ilyapopovs** [comment]
Tested on:
Firefox 105.0.3 ✅ 
Chrome 106.0.5249.119 ✅ 

But I might have missed some small part of the UI.
Would be nice if someone else could validate as well :)

**Comment by nguyenkims** [comment]
@ilyapopovs thanks for making the PR and sorry for the late reply! I tested the PR and it works well. Our developers are currently reviewing the code and will make sure to come back to you quickly!

**Comment by edvincandon** [comment]
@ilyapopovs Looks good to me 👍🏻
I've pushed a commit on your PR moving every theme utilities into a single place.

**Comment by ilyapopovs** [comment]
@edvincandon Changes look good to me 👍 

Let me know if anything else is needed from my side to move this forward :)

**Review by nguyenkims — APPROVED** [review]
LGTM, thanks for making this feature!

**Comment by ilyapopovs** [comment]
Glad to help! 🙂 

Also, Issue https://github.com/simple-login/browser-extension/issues/118 can be closed

**Comment by nguyenkims** [comment]
@ilyapopovs we are going to make a release for this version and will announce it on our social media. If that works for you, we'd like to mention you in these posts as this feature is highly requested and it only becomes available thanks to your contribution. In this case please send me your social media handle. SL is active on Twitter, Reddit and Linkedin. cc @alext

**Comment by ilyapopovs** [comment]
@nguyenkims sure!
I'm not particularly active on Twitter or Reddit, but it's `@ilyapopovs` on those 3.

**Pull request merged**